queries/systemverilog/highlights.scm

```diff
@@ -0,0 +1,528 @@
+  "type"
+  "void"
+  "coverpoint"
+  "cross"
+  "nettype"
+  "export"
+  "force"
+  "release"
+  "timeunit"
+  "timeprecision"
+  "sequence"
+  "endsequence"
+  "property"
+  "endproperty"
+  "clocking"
+  "endclocking"
+  "iff"
+  "covergroup"
+  "endgroup"
+  "specify"
+  "endspecify"
+  "primitive"
+  "endprimitive"
+  "wait"
+  "wait_order"
+  "const"
+  "constraint"
+  "unique"
+  "do"
+  "genvar"
+  "inside"
+  "rand"
+  "new"
+  "continue"
+  "randc"
+  (unique_priority)
+  (bins_keyword)
+] @keyword
+
+[
+  "class"
+  "endclass"
+  "interface"
+  "endinterface"
+  "enum"
+  "struct"
+  "union"
+] @keyword.type
+
+[
+  "function"
+  "endfunction"
+  "task"
+  "endtask"
+] @keyword.function
+
+"return" @keyword.return
+
+[
+  (always_keyword)
+  "for"
+  "foreach"
+  "repeat"
+  "forever"
+  "initial"
+  "while"
+] @keyword.repeat
+
+[
+  "if"
+  "else"
+  (case_keyword)
+  "endcase"
+] @keyword.conditional
+
+[
+ "?"
+ ":"
+ "#"
+] @keyword.conditional.ternary
+
+[
+  "="
+  "-"
+  "+"
+  "/"
+  "*"
+  "^"
+  "&"
+  "|"
+  "&&"
+  "||"
+  "{"
+  "}"
+  "'{"
+  "<="
+  "=="
+  "!="
+  "==="
+  "!=="
+  "-:"
+  "<"
+  ">"
+  ">="
+  "%"
+  ">>"
+  "<<"
+  "|="
+  "|=>"
+  "|->"
+  ">>>"
+  "<<<"
+  "->>"
+  "->"
+  (unary_operator)
+  (inc_or_dec_operator)
+  (queue_dimension)
+] @operator
+
+[
+  ";"
+  "::"
+  ","
+  "."
+] @punctuation.delimiter
+
+[
+  "["
+  "]"
+  "("
+  ")"
+] @punctuation.bracket
+
+[
+  "or"
+  "and"
+] @keyword.operator
+
+(cast
+  [
+    "'"
+    "("
+    ")"
+  ] @operator)
+
+[
+  "input"
+  "output"
+  "inout"
+  "signed"
+  "unsigned"
+  "assert"
+  "cover"
+  "assume"
+  "disable"
+  "automatic"
+  "static"
+  (dpi_function_import_property)
+  (dpi_task_import_property)
+] @keyword.modifier
+
+[
+ "include"
+ "import"
+] @keyword.import
+
+(comment) @comment @spell
+
+port_name: (simple_identifier) @variable
+
+(port
+  (simple_identifier) @variable)
+
+(list_of_port_identifiers
+  (simple_identifier) @variable)
+
+(net_lvalue
+  (simple_identifier) @variable)
+
+(clocking_event) @attribute
+
+(cycle_delay_range) @attribute
+
+(delay_control) @attribute
+
+(cycle_delay) @attribute
+
+(timescale_compiler_directive) @attribute
+
+(time_literal) @attribute
+
+(attribute_instance) @attribute
+
+(attribute_instance
+  (attr_spec
+    (simple_identifier) @property))
+
+[
+  (integral_number)
+  (unbased_unsized_literal)
+  (fixed_point_number)
+] @number
+
+[
+  (net_type)
+  (integer_vector_type)
+  (integer_atom_type)
+] @type.builtin
+
+
+(list_of_variable_decl_assignments
+  (variable_decl_assignment
+    name: (simple_identifier) @variable))
+
+(hierarchical_identifier
+  (simple_identifier) @variable)
+
+(simulation_control_task) @function.builtin
+
+(system_tf_identifier) @function.builtin
+
+(severity_system_task) @function.builtin
+
+(string_literal
+  (quoted_string) @string)
+
+(include_compiler_directive) @keyword.import
+
+(include_compiler_directive
+  (quoted_string) @string.special.path)
```

Comment:
I think `(system_lib_string)` should also be handled (probably just as `@string`)